presto-iceberg/src/test/java/com/facebook/presto/iceberg/nessie/TestIcebergSystemTablesNessie.java

```diff
@@ -90,15 +90,16 @@ public void testPropertiesTable()
     {
         assertQuery("SHOW COLUMNS FROM test_schema.\"test_table$properties\"",
                 "VALUES ('key', 'varchar', '', '')," + "('value', 'varchar', '', '')");
-        assertQuery("SELECT COUNT(*) FROM test_schema.\"test_table$properties\"", "VALUES 2");
+        assertQuery("SELECT COUNT(*) FROM test_schema.\"test_table$properties\"", "VALUES 3");
         List<MaterializedRow> materializedRows = computeActual(getSession(),
                 "SELECT * FROM test_schema.\"test_table$properties\"").getMaterializedRows();
 
-        // nessie writes a "nessie.commit.id" to the table properties
-        assertThat(materializedRows).hasSize(2);
+        // nessie writes a "nessie.commit.id" + "gc.enabled=false" to the table properties
+        assertThat(materializedRows).hasSize(3);
```

Comment:
Just for confirmation: You updated the size from 2 to 3. Is it because you are adding `gc.enabled=false` to the properties?

Reply:
yes exactly

Reply:
Thanks for the clarification!